crates/holochain/src/sweettest/sweet_conductor_batch.rs

```diff
@@ -104,6 +105,73 @@ impl SweetConductorBatch {
             .collect::<Result<Vec<_>, _>>()?
             .into())
     }
+}
+
+#[cfg(feature = "no-hash-integrity")]
+use holochain_p2p::*;
+#[cfg(feature = "no-hash-integrity")]
+use kitsune_p2p::test_util::scenario_def::{PeerMatrix, ScenarioDef};
+
+#[cfg(feature = "no-hash-integrity")]
+impl SweetConductorBatch {
```

Comment:
It feels a little weird to have to put these "no-hash-integrity" checks on all these Sweettest items that presumably can't really function without this feature...

Would it make more sense structurally if sweettest was a separate crate from holochain?

Reply:
@neonphog I would like sweettest to be a separate crate, but we already have a bunch of tests written in it in the holochain crate. So it would take a bit of work.

Note that this is not the only impl, the other impl above this one is not behind a feature. But other than that I don't get your comment, because indeed these methods can't function without this feature, which is why the feature is on them...